test.py

```diff
@@ -37,6 +37,7 @@
 from io import StringIO
 from scripts import coverage    # type: ignore
 from test.pylib.artifact_registry import ArtifactRegistry
+from test.pylib.cpp.ldap.prepare_instance import try_something_backoff, can_connect, setup
```

Comment:
I don't anything against this patch specific, but I don't know how to test it (I hope one of the LDAP developers - who is this by the way?) can review this patch, but I want to comment that such a patch doesn't really belong in a series supposedly about Boost and unit tests. Maybe you can at least change the cover letter and title to say "... and LDAP tests".